spec/views/groups/index.html.haml_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "groups/index", type: :view do
+  before(:each) do
+    assign(:groups, [
+      Group.create!(
+        :category => "Category",
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.